setup.py

```diff
@@ -76,6 +76,7 @@ def is_pure(self):
         "tensorflow-datasets",
         "keras-core",
         "kagglehub",
+        "keras_nlp"
```

Comment:
```suggestion
        "keras_nlp",
```

Also, rebasing on top of current master should fix the CI.